lib/main.dart

```diff
@@ -56,3 +46,11 @@ Future<void> main({AppConfig appConfig = scAppConfig}) async {
     runApp(SchulCloudApp());
   });
 }
+
+void _showLoadingPage() {
+  runApp(Container(
+    color: Colors.white,
```

Comment:
I'll just point out that this will be quite jarring to people using dark mode.
How does the `MaterialApp` figure out if the system is in dark mode or not? We should probably do the same here.